c_impl: args parsing type issue

After making it build, i thought i better run the tests (on Apple M1):
```
_________________________________________________________ test_usedforsecurity_ignored _________________________________________________________

    def test_usedforsecurity_ignored():
>       blake3(usedforsecurity=True)
E       ValueError: invalid value for max_threads

tests/test_blake3.py:360: ValueError
=========================================================== short test summary info ============================================================
FAILED tests/test_blake3.py::test_usedforsecurity_ignored - ValueError: invalid value for max_threads
```

--- a/c_impl/blake3module.c
+++ b/c_impl/blake3module.c
@@ -53,7 +53,7 @@ static PyObject *Blake3_new(PyTypeObject *type, PyObject *args,
   Py_buffer key = {0};
   const char *derive_key_context = NULL;
   Py_ssize_t max_threads = 1;
-  bool usedforsecurity = true;
+  int usedforsecurity = 1;
 
   PyObject *ret = NULL;
 

